transaction/src/action.rs

```diff
@@ -55,9 +54,10 @@ impl Action {
             Action::Spend(spend) => spend.body.value_commitment,
             Action::Delegate(delegate) => delegate.value_commitment(),
             Action::Undelegate(undelegate) => undelegate.value_commitment(),
-            // TODO: re-enable when Swap/SwapClaim is ready
-            // Action::Swap(swap) => swap.value_commitment(),
-            // Action::SwapClaim(swap_claim) => swap_claim.value_commitment(),
+            Action::Swap(swap) => swap.value_commitment(),
+            // A SwapClaim action has no impact on the transaction's value balance as the
+            // outputs are handled internally to the action.
+            Action::SwapClaim(_) => value::Commitment::default(),
```

Comment:
I don't think this is quite right -- the `SwapClaim` *contributes* the fee amount that was prepaid in the `Swap`, so that it can be consumed by the transaction's fee.